network/bitvector/bitvector.go

```diff
@@ -57,6 +64,38 @@ func (bv *BitVector) Set(i int, v bool) {
 	}
 }
 
+// SetBytes modifies all bits in the bitvector that are set in the argument
+//
+// If v is true, it sets all bits in the bitvector that are set in the argument
+// If v is false, it unsets all bites in the bitvector that are set in the argument
```

Comment:
typo - `bites` -> `bits`.

`Unset 1010101001` is easier to reason about than `SetBytes 1010101001 false` IMO.

Reply:
I changed it because the corresponding one for individual bits is `Set(int, bool)` - should we change that too,  then?